src/Microsoft.DotNet.Build.Tasks/Targets/packages.targets

```diff
@@ -61,7 +61,7 @@
 
     <GetPackageVersion
       Condition="Exists('$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll')"
-      DailyBuildNumber="$(DailyBuildNumber)"
+      BuildNumber="$(BuildNumber)"
```

Comment:
Should we default this to something if it isn't set? To avoid errors when people try to use this without having set it?